### ISSUE TITLE
[BUG] Graph symbols not rendering at uniform width in "Secure Shell App", "hterm"

### ISSUE THREAD

**Issue by turkeyphant** [aristocratos/bashtop#159]
**Describe the bug**

![image](https://user-images.githubusercontent.com/8129583/86477959-5b451780-bd41-11ea-9039-3dc135fb0d97.png)

The graph "pushes" the cores to the right:

![image](https://user-images.githubusercontent.com/8129583/86478136-b2e38300-bd41-11ea-8643-dbf9b8c7b349.png)

**To Reproduce**

run bashtop and leave running.

**Expected behavior**

![image](https://user-images.githubusercontent.com/8129583/86477984-6b5cf700-bd41-11ea-9b9a-eb2c74f1f3e0.png)

**Info (please complete the following information):**
Bashtop version: 0.8.29
Linux distribution and version: Linux 4.19.107-Unraid.
Terminal used: Secure Shell App
Font used: Source Code Pro
Bash version, "bash --version" (version 4.4 or above is required): 5.0.11(1)-release (x86_64-slackware-linux-gnu)
Locales: output of "locale -v"
LANG=en_US.UTF-8
LC_CTYPE="en_US.UTF-8"
LC_NUMERIC="en_US.UTF-8"
LC_TIME="en_US.UTF-8"
LC_COLLATE=C
LC_MONETARY="en_US.UTF-8"
LC_MESSAGES="en_US.UTF-8"
LC_PAPER="en_US.UTF-8"
LC_NAME="en_US.UTF-8"
LC_ADDRESS="en_US.UTF-8"
LC_TELEPHONE="en_US.UTF-8"
LC_MEASUREMENT="en_US.UTF-8"
LC_IDENTIFICATION="en_US.UTF-8"
LC_ALL=

**Additional context**

contents of `$HOME/.config/bashtop/error.log`

```
New instance of bashtop version: 0.8.29 Pid: 25899
 
New instance of bashtop version: 0.8.29 Pid: 25783
 
New instance of bashtop version: 0.8.29 Pid: 23471
 
New instance of bashtop version: 0.8.29 Pid: 360
```

**Comment by turkeyphant** [comment]
Just upgraded to 0.9.19 and bug still persists. Window size is 131 * 41.

**Comment by aristocratos** [comment]
This isn't a issue caused by bashtop, but by the terminal you are using. Make a bug report that some symbols aren't rendered correctly with the creator of the terminal.

**Comment by turkeyphant** [comment]
Which symbols are being rendered incorrectly so I can report this?

**Comment by aristocratos** [comment]
> Also needs a UTF8 locale and a font that covers:
> 
>     Unicode Block “Braille Patterns” U+2800 - U+28FF
>     Unicode Block “Geometric Shapes” U+25A0 - U+25FF
>     Unicode Block "Box Drawing" and "Block Elements" U+2500 - U+259F

Looks to be the "Braille Patterns" block that's not being rendered at a uniform width.

**Comment by aristocratos** [comment]
@turkeyphant 
I don't think that font is at fault here since it works fine in a regular terminal.
I think the issue is with the terminal itself:

> Terminal used: Secure Shell App

**Comment by turkeyphant** [comment]
> @turkeyphant
> I don't think that font is at fault here since it works fine in a regular terminal.
> I think the issue is with the terminal itself:
> 
> > Terminal used: Secure Shell App

It seems the font doesn't contain the glyphs so it's the fallback font that doesn't have uniform width glyphs. I'd tried every font available including Cousine, Inconsolata, Roboto Mono, Menlo, etc. and all have the same issue.

I'm not sure if it's all the fonts at issue but would it be possible to have an option to use alternative glyphs that are more widely supported?

**Comment by aristocratos** [comment]
Since the symbols of the fallback font is correct it's far more likely it's the terminal that hasn't got proper wide character support.

And I'm not gonna spend time coding to accommodate for bugs in other software, especially when I don't even know which symbols would render correctly.

The way I see it you got 4 alternatives:
1. Read through the [documentation](https://chromium.googlesource.com/apps/libapps/+/HEAD/hterm/doc/) for [hterm](https://hterm.org/) and see if you can find any special options for font rendering that could help you.
2. Report the bug and get support for the issue at https://bugs.chromium.org/p/chromium/issues/list?can=2&q=component%3APlatform%3EApps%3EDefault%3EHterm
3. Use another terminal emulator.
4. Use another resource monitor.

**Comment by turkeyphant** [comment]
Fair enough.

**Comment by aristocratos** [comment]
I will reopen this to bring attention to it in the hopes somebody else has a solution.

And I would urge you to share if you find a solution.
